src/csharp/Grpc.Core.Tests/CallCancellationTest.cs

```diff
@@ -189,5 +189,41 @@ public void CanDisposeDefaultCancellationRegistration()
 
             using (obj) {}
         }
+
+        [Test]
+        public async Task ServerStreamingCall_CancelAfterServerHasFinished()
+        {
+            // Test to verify the fix for https://github.com/grpc/grpc/issues/17761 that was
+            // fixed in c-core https://github.com/grpc/grpc/issues/18234
```

Comment:
https://github.com/grpc/grpc/issues/18234 is an issue and from reading its comment thread it's not clear to me which was the PR that fixed the problem. If https://github.com/grpc/grpc/issues/18234 has really been fixed, it would be good to point out with PR fixed it (and close it as such).

Reply:
Probably wasn't https://github.com/grpc/grpc/issues/18234 but was fixed sometime between tags v1.30.0 - June 2020 (broken) and v1.31.0-pre1 -July 2020 (fixed).  I haven't managed to narrow down with a git bisect as my dev environment is too different from the older builds.